test/BaselineOutput/Common/EntryPoints/core_manifest.json

```diff
@@ -6622,7 +6622,7 @@
           "Default": "GradientDescent"
         },
         {
-          "Name": "EarlyStoppingRule",
+          "Name": "EarlyStoppingRuleFactory",
           "Type": {
             "Kind": "Component",
             "ComponentKind": "EarlyStoppingCriterion"
```

Comment:
>"ComponentKind": "EarlyStoppingCriterion" [](start = 12, length = 41)

Manifest is broken. Here is a reference to ComponenKInd "EarlyStoppingCriterion" but the Components for this component kind is removed from manifest. This would  Entrypoint Compiler to fail to figure out fitting components during entrypoint graph parse

Reply:
No (at least at iteration 3), those definitions of EarlyStoppingCriterion are still there. Please try searching for "EarlyStoppingCriterion".

---
In reply to: [262645293](https://github.com/dotnet/machinelearning/pull/2851#discussion_r262645293) [](ancestors = 262645293)